eng/scripts/Sdk-Generate.ps1

```diff
@@ -0,0 +1,144 @@
+param(
+  [string] $InputJsonPath = $GenerateInput,
+  [string] $OutputJsonPath = $GenerateOutput,
```

Comment:
Where `GenerateInput` and `GenerateOutput` are expected to come from?

Reply:
[GenerateInput ](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=107) and [GenerateOutput](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=270) is passed in [SDK Automation step](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5). It is configured in the pipeline.